specification/src/main/asciidoc/platform/PlatformOverview.adoc

```diff
@@ -893,7 +893,7 @@ well. See the individual API specifications for details. Finally, J2EE
 === Changes in J2EE 1.4
 
 The primary focus of J2EE 1.4 is support for
-web services. The JAX-RPC and SAAJ APIs provide the basic web services
```

Comment:
Not sure JAX-RPC should be changed here as it is talking about an old version

Reply:
Agree, don't change since it's referring to J2EE 1.4 content.